src/app/components/module-index-page/module-index-page.component.html

```diff
@@ -93,11 +93,11 @@
                                     <!-- Drag and Drop List of Content -->
 
                                     <div cdkDropList (cdkDropListDropped)="onDrop($event, module)">
-                                        <div class="link-box lead" *ngFor="let link of module.links | contentOrder: module : 1; let i = index" cdkDrag
+                                        <div class="link-box lead" *ngFor="let link of module.links; let i = index" cdkDrag
                                             [ngClass]="{'active': contentActive.get(module) === i}" (click)="setActiveContent(module, i)">
                                             <em style="width:20px;" class="fa" [ngClass]="{'fa-file-code-o': link.content.format === 'Code', 'fa-file-text-o': link.content.format === 'Document', 'fa-file-powerpoint-o': link.content.format === 'Powerpoint'}"></em>
 
-                                            &nbsp; {{link.content.title}}
+                                            &nbsp;{{link.priority}} - {{link.content.title}}
```

Comment:
Why add this explicitly?

Reply:
forgot to remove it when testing to make sure it functioned without the pipe